test/tests/functional/pbs_init_script.py

```diff
@@ -43,23 +43,28 @@ class TestPbsInitScript(TestFunctional):
     """
     Testing PBS Pro init script
     """
+
     def test_env_vars_precede_pbs_conf_file(self):
         """
         Test PBS_START environment variables overrides values in pbs.conf file
         """
-        self.du.run_cmd(cmd=['/etc/init.d/pbs', 'stop'])
-
         conf = {'PBS_START_SERVER': '1', 'PBS_START_SCHED': '1',
-                'PBS_START_COMM': '1', 'PBS_START_MOM': '0'}
+                'PBS_START_COMM': '1', 'PBS_START_MOM': '1'}
+        self.du.set_pbs_config(confs=conf)
+
+        self.du.run_cmd(cmd=['/etc/init.d/pbs', 'stop'], sudo=True)
+
+        conf['START_PBS_MOM'] = '0'
```

Comment:
Is START_PBS_MOM a typo? should it be PBS_START_MOM?

Reply:
Good catch.  I tried to figure out why this test passed with this typo, and then I realized that PBS_START_MOM=1 from pbs.conf and worked fine by not being overridden.  I've corrected the typo.